### PR TITLE
Beta react developer tools translation done

### PR DESCRIPTION
**Pull request by Anishpras**

### CODE REVIEW

**Comment by Anishpras** [comment]
@arshadkazmi42 I have translated the page. Can you please review it?

File: beta/src/pages/learn/react-developer-tools.md [hunk 1]
```diff
@@ -30,26 +29,26 @@ yarn global add react-devtools
...
-Then connect your website by adding the following `<script>` tag to the beginning of your website's `<head>`:
+फिर अपनी वेबसाइट के `<head>` की शुरुआत में निम्न `<script>` टैग जोड़कर अपनी वेबसाइट कनेक्ट करें:
 ```html {3}
 <html>
   <head>
     <script src="http://localhost:8097"></script>
 ```
 
-Reload your website in the browser now to view it in developer tools.
+अपनी वेबसाइट को डेवलपर टूल में देखने के लिए उसे अभी ब्राउज़र में पुनः लोड करें।
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/react-developer-tools.md:44]: `पुनः लोड` => `रीलोड`

let me know if you have any other better suggestion for this.

File: beta/src/pages/learn/react-developer-tools.md [hunk 2]
```diff
@@ -30,26 +29,26 @@ yarn global add react-devtools
...
 
-![React Developer Tools standalone](/images/docs/react-devtools-standalone.png)
+![रिएक्ट डेवलपर टूल्स स्टैंडअलोन](/images/docs/react-devtools-standalone.png)
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/react-developer-tools.md:46]: `रिएक्ट` => `React`

File: beta/src/pages/learn/react-developer-tools.md [hunk 3]
```diff
@@ -30,26 +29,26 @@ yarn global add react-devtools
...
 
-## Mobile (React Native) {/*mobile-react-native*/}
-React Developer Tools can be used to inspect apps built with [React Native](https://reactnative.dev/) as well.
+## मोबाइल (React Native) {/*mobile-react-native*/}
+रिएक्ट डेवलपर टूल का उपयोग [React Native](https://reactnative.dev/) के साथ निर्मित ऐप्स का निरीक्षण करने के लिए भी किया जा सकता है।
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/react-developer-tools.md:49]: `रिएक्ट` => `React`

change this everywhere

Comment (arshadkazmi42) [inline beta/src/pages/learn/react-developer-tools.md:49]: `निर्मित` => `बनी`

File: beta/src/pages/learn/react-developer-tools.md [hunk 4]
```diff
@@ -30,26 +29,26 @@ yarn global add react-devtools
...
 
-The easiest way to use React Developer Tools is to install it globally:
+रिएक्ट डेवलपर टूल्स का उपयोग करने का सबसे आसान तरीका इसे ग्लोबली इनस्टॉल  करना है:
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/react-developer-tools.md:51]: `रिएक्ट` => `React`

File: beta/src/pages/learn/react-developer-tools.md [hunk 5]
```diff
@@ -58,13 +57,13 @@ yarn global add react-devtools
...
 ```
 
-Next open the developer tools from the terminal.
+इसके बाद टर्मिनल से डेवलपर टूल खोलें।
 ```bash
 react-devtools
 ```
 
-It should connect to any local React Native app that's running.
+इसे चल रहे किसी भी स्थानीय React Native ऐप से कनेक्ट होना चाहिए।
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/react-developer-tools.md:65]: `स्थानीय` => `लोकल`

File: beta/src/pages/learn/react-developer-tools.md [hunk 6]
```diff
@@ -58,13 +57,13 @@ yarn global add react-devtools
...
 
-> Try reloading the app if developer tools doesn't connect after a few seconds.
+> यदि डेवलपर टूल कुछ सेकंड के बाद कनेक्ट नहीं होते हैं, तो ऐप को पुनः लोड करने का प्रयास करें।
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/react-developer-tools.md:67]: `पुनः लोड` => `रीलोड`

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
Great work.

Added review once these are fixed its good to be merged.

**Comment by Anishpras** [comment]
@arshadkazmi42 I have done the changes. Thank you.

**Review by arshadkazmi42 — APPROVED** [review]
Looks good :tada: 

Lets get this merged.

**Pull request merged**